supervisor/backups/manager.py

```diff
@@ -147,8 +145,9 @@ async def _do_backup(
                     await backup.store_addons(addon_list)
 
                 # Backup folders
-                if FOLDER_HOMEASSISTANT in folder_list:
-                    await backup.store_homeassistant_config_dir()
+                # HomeAssistant Folder is for v1
+                if FOLDER_HOMEASSISTANT in folder_list or homeassistant:
```

Comment:
```suggestion
                if homeassistant or FOLDER_HOMEASSISTANT in folder_list:
```
Checking a bool is faster than checking items in a list